BootloaderCorePkg/Stage2/Stage2Support.c

```diff
@@ -122,6 +122,79 @@ DisplaySplash (
   return Status;
 }
 
+/**
+  Initialize the virtual console framebuffer back-end.
+
+  @retval EFI_UNSUPPORTED Could not initilaize the console.
+  @retval EFI_SUCCESS     Console initialized.
+**/
+EFI_STATUS
+EFIAPI
+ConsoleInitFrameBuffer (
```

Comment:
I think Output console init needs to be in ConsoleOutLib instead of Stage2 core code.